controller/environment.go

```diff
@@ -18,13 +18,15 @@ const (
 
 type EnvironmentController struct {
 	*goa.Controller
-	db application.DB
+	db                   application.DB
+	developerModeEnabled bool // TODO remove this option
 }
 
-func NewEnvironmentController(service *goa.Service, db application.DB) *EnvironmentController {
+func NewEnvironmentController(service *goa.Service, db application.DB, developerModeEnabled bool) *EnvironmentController {
```

Comment:
Pass configuration interface instead. For example - https://github.com/fabric8-services/fabric8-cluster/blob/master/controller/status.go#L41
https://github.com/fabric8-services/fabric8-cluster/blob/master/controller/status.go#L56

Reply:
this is temp, so any way should be fine :-)

Reply:
Most likely you will need configuration later anyway.

Reply:
In future, I will follow config_way but for this time it's already done and also for time being, let it be ;-)